guide/14-deep-learning/exploring_model_extensibility_for_named_entity_recognition.ipynb

```diff
@@ -0,0 +1,722 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        "ArcGISLearnVersion": "2.4.0"
</pre><p>Is this key important, the version will keep on changing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Yes and provided the information. </p>